randomize/randomize.go

```diff
@@ -52,7 +52,7 @@ var (
 	rgxValidTime     = regexp.MustCompile(`[2-9]+`)
 
 	validatedTypes = []string{
-		"inet", "line", "uuid", "interval", "mediumint",
+		"inet", "line", "uuid", "uniqueidentifier", "interval", "mediumint",
```

Comment:
Is it possible to coerce the value of uniqueidentifier to uuid to make it simpler to deal with?